bench/tx-generator/src/Cardano/Benchmarking/Script/Types.hs

```diff
@@ -18,7 +18,7 @@ import           Data.List.NonEmpty
 import           Data.Text (Text)
 
 import           Cardano.Api (AnyCardanoEra, ExecutionUnits, Lovelace, ScriptData, ScriptRedeemer,
-                   TextEnvelope, TxIn)
+                   TextEnvelope, TxIn, WithCBOR)
```

Comment:
That should be `HashableScriptData` - I must have missed fixing it after introducing the `HashableScriptData` type synonym